.azure-pipelines/client.yml

```diff
@@ -0,0 +1,59 @@
+trigger:
+- master
+
+jobs:
+- job: 'Build_Target_Packages'
+
+  pool:
+    vmImage: 'ubuntu-16.04'  
+
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.6'
+    inputs:
+      versionSpec: 3.6
+
+  - script: |
+     pip install wheel
+
+    displayName: 'Prep Environment, Install Packages'
+
+  - powershell: |
+     $inputFilterSet="$(build_targeting_string)"
+     
+     if ($inputFilterSet -eq "")
+     {
+         $inputFilterSet = "azure-*"
```

Comment:
minor but at some point we may want to change how we do initialization of these variables and move them into the hosting build pipeline.

Reply:
Yeah. When we settle, I will update this.